features/networking/multus.feature

```diff
@@ -424,10 +424,14 @@ Feature: Multus-CNI related scenarios
       | n | <%= project.name %>                                                                                                                              |
     Then the step should succeed
     #Creating no-vlan pod absorbing above net-attach-def
+    Given I store all worker nodes to the :nodes clipboard
+    And CNI vlan info is obtained on the "<%= cb.nodes[0].name %>" node
+    # force pod onto the node we already got CNI vlan information for
     When I run oc create over "<%= BushSlicer::HOME %>/testdata/networking/multus-cni/Pods/generic_multus_pod.yaml" replacing paths:
       | ["metadata"]["name"] | pod-novlan |
```

Comment:
@rbbratta while you are at it, pls align the tables in PR.  thx!

Reply:
fixed.   

there were lots of whitespace problems.  we should do a `sed -i -e 's/[[:space:]]\+$//g'` or similar to clean all the trailing whitespace.

is it possible to do a linting step with the PR robot?  I've done travis CI with github once, a while ago.

Reply:
Okay, I let RubyMine fixup all the tables, hopefully that should fix it.